test/Makefile

```diff
@@ -0,0 +1,15 @@
+CC = gcc
+CFLAGS = -Wall -Wextra -std=c11
+LIBS = -lcheck -pthread -lrt -lm
+
+test: test_test.o runner.o
+	$(CC) $(CFLAGS) -o test test_test.o runner.o $(LIBS)
```

Comment:
test is probably not a good name for the binary - do `pgmoneta_ext_test` or something